### PR TITLE
Update uvtools.rb: Add live check and dependencies

### PR DESCRIPTION
**Pull request by sn4k3**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/uvtools.rb [hunk 1]
```diff
@@ -10,6 +10,15 @@
...
   homepage "https://github.com/sn4k3/UVtools"
 
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
+
+  auto_updates true
+  depends_on macos: ">= :catalina"
+  depends_on formula: "mono-libgdiplus"
```
Comment (miccal) [inline Casks/uvtools.rb:20]: Why is this dependency required for the `app`? This is only required when building from source. no?

Comment (sn4k3) [reply Casks/uvtools.rb:20]: App runs fine without `mono-libgdiplus` but a little function (rarely used by users) require that lib in order to run (**Export layers to GIF**). If not installed app will not crash but will show an error message that a type is missing and prevent the execution of the tool, it's really not critical. You think it's best to remove it so it not bloat the install and users who want to execute that i can instruct them to install `mono-libgdiplus`?
(Also not required to build)

image

image

Comment (miccal) [reply Casks/uvtools.rb:20]: `mono-libgdiplus` is a heavy dependency, so I think it is best not to add it:
```
|-> brew info mono-libgdiplus
==> mono-libgdiplus: stable 6.1 (bottled)
GDI+-compatible API on non-Windows operating systems
https://www.mono-project.com/docs/gui/libgdiplus/
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/mono-libgdiplus.rb
License: MIT
==> Dependencies
Build: pkg-config ✔
Required: cairo ✔, fontconfig ✔, freetype ✔, gettext ✔, giflib ✔, glib ✔, jpeg-turbo ✔, libexif ✘, libpng ✔, libtiff ✔, pango ✔
==> Analytics
install: 2,009 (30 days), 7,489 (90 days), 21,406 (365 days)
install-on-request: 2,012 (30 days), 7,492 (90 days), 21,410 (365 days)
build-error: 0 (30 days)
```

Comment (sn4k3) [reply Casks/uvtools.rb]: That's ok for me 👍

**Review by miccal — APPROVED** [review]

**Comment by sn4k3** [comment]
@miccal One question, i notice livecheck has been removed on your [commit](https://github.com/Homebrew/homebrew-cask/pull/133600/commits/43f860112a66f2a89096dce5a3bc7ba789b50a46) ?

**Comment by miccal** [comment]
Yes, for GitHub releases the `livecheck` is automatic and not needed.

**Comment by sn4k3** [comment]
> Yes, for GitHub releases the `livecheck` is automatic and not needed.

Ok, didn't know about that. Thanks!

**Comment by sn4k3** [comment]
@miccal Sorry to bring the topic here but I have a few questions to see if i can improve the deployment of UVtools to the ease of use.

My app is not codesigned, this means:

- Security flag to trigger and users must go security and allow it to run (some users fail to do this), on my auto installer script I bypass that by [removing the quarantine attribute](https://github.com/sn4k3/UVtools/blob/843c3e817e812cfb291c24d65063ee9e099e4f63/Scripts/install-uvtools.sh#L56).  
**Question:** Is possible to run such line on the post install cask or it is against the deployment rules?
- `arm64` app doesn't run due not codesign app. I bypass that by [creating a run file on user desktop](https://github.com/sn4k3/UVtools/blob/843c3e817e812cfb291c24d65063ee9e099e4f63/Scripts/install-uvtools.sh#L59-L85) it end in calling `dotnet UVtools.dll` instead which is able to run the app just fine.  
**Question:** Is possible to do the same on this cask if arm64? Also it would require `dotnet-sdk` dependency on `arm64` only

**Comment by miccal** [comment]
> @miccal Sorry to bring the topic here but I have a few questions to see if i can improve the deployment of UVtools to the ease of use.
> 
> My app is not codesigned, this means:
> 
> * Security flag to trigger and users must go security and allow it to run (some users fail to do this), on my auto installer script I bypass that by [removing the quarantine attribute](https://github.com/sn4k3/UVtools/blob/843c3e817e812cfb291c24d65063ee9e099e4f63/Scripts/install-uvtools.sh#L56).
>   **Question:** Is possible to run such line on the post install cask or it is against the deployment rules?

We do not allow this, but users can install with the `--no-quarantine` flag if they wish.

> * `arm64` app doesn't run due not codesign app. I bypass that by [creating a run file on user desktop](https://github.com/sn4k3/UVtools/blob/843c3e817e812cfb291c24d65063ee9e099e4f63/Scripts/install-uvtools.sh#L59-L85) it end in calling `dotnet UVtools.dll` instead which is able to run the app just fine.
>   **Question:** Is possible to do the same on this cask if arm64? Also it would require `dotnet-sdk` dependency on `arm64` only

I am afraid that we would not allow this either -- we do not bypass the built-in macOS security features in Casks.

**Pull request merged**